### PR TITLE
Andrew bastian

### PR DESCRIPTION
**Pull request by andrewbastian**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1,153 @@
...
+})
+
+// create a new user
+server.post('/api/users', (req, res) => {
+    const {
+        name,
+        bio
+    } = req.body
+    if (!name || !bio) {
+        return res.status(400).json({
```
Comment (Derrick-Mei) [inline index.js]: I wonder, since you're using an if/else block.  Will it still work if you don't have the return statement?

File: index.js [hunk 2]
```diff
@@ -1 +1,153 @@
...
+    if (!name || !bio) {
+        res.status(400).json({
+            errorMessage: 'Please provide name and bio for the user.'
+        })
+    } else {
+        db.update(req.params.id, req.body)
+            .then(user => {
+                if (user) {
+                    res.status(200).json({
+                        message: `user ${id} name changed to ${name} and bio to ${bio}`
```
Comment (Derrick-Mei) [inline index.js]: Nice job with your return message!  Very descriptive

File: index.js [hunk 3]
```diff
@@ -1 +1,153 @@
...
+    }
+})
+
+// delete user
+
+server.delete('/api/users/:id', (req, res) => {
+
+    db.remove(req.params.id)
+        .then(response => {
+            if (response == 1) {
```
Comment (Derrick-Mei) [inline index.js]: 2 things, first is that `==` is generally a lesser accepted answer to check for equality.  You should stick to the `===` although it's unlikely that the database is going to return a string `'1'`.  

Second, it's probably not likely that the database returns more than 1 deleted result at a time, but in the event that it does `>= 1` would be a better option

File: package.json [hunk 4]
```diff
@@ -4,7 +4,8 @@
   "description": "Node API 1 Project Starter Code",
   "main": "index.js",
   "scripts": {
-    "server": "nodemon index.js"
+    "server": "nodemon index.js",
+    "start": "node index.js"
```
Comment (Derrick-Mei) [inline package.json]: good job adding the start script.  You can add nodemon here too if you like.

**Review by liam-mei — APPROVED** [review]

**Comment by liam-mei** [comment]
## MVP

- [x] Post New User
- [x] Get All Users
- [x] Get User By Id
- [x] Delete User By Id

## Stretch

- [x] Update User By Id
- [ ] Implement CORS + React Front-End
--------------------
score: 3
--------------------

**Pull request merged**